docs/_functions/domain/CAA.md

```diff
@@ -0,0 +1,34 @@
+---
+name: CAA
+parameters:
+  - name
+  - tag
+  - value
+  - modifiers...
+---
+
+CAA adds a CAA record to a domain. The name should be the relative label for the record. Use `@` for the domain apex.
+
+Tag can be one of "issue", "issuewild" or "iodef". Value depends on the tag, and is automatically quoted.
+
```

Comment:
The target field doesn't always have to be a domain; the meaning is depending on which tag is used.

For example, issue/issuewild also accept an optional parameter: `issue "example.net; acme-methods=dns-01,http-01"`. iodef accept either http/https URL or mailto URL.

I think it is ok to leave target unparsed, except for automatic quoting (speaking of which, I haven't add escaping for `"`. I'll have to add that)